src/Test/Utilities/Portable/Compilation/OperationTreeVerifier.cs

```diff
@@ -1758,7 +1792,8 @@ public override void VisitPatternCaseClause(IPatternCaseClauseOperation operatio
             Assert.Same(((ICaseClauseOperation)operation).Label, operation.Label);
 
             Visit(operation.Pattern, "Pattern");
-            Visit(operation.Guard, "Guard Expression");
+            if (operation.Guard != null)
```

Comment:
> if (operation.Guard != null) [](start = 11, length = 29)

Is this conditional logic really necessary?

Reply:
It is helpful in making the output more readable.

---
In reply to: [244206384](https://github.com/dotnet/roslyn/pull/32018#discussion_r244206384) [](ancestors = 244206384)